guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -261,6 +261,31 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
     };
 
     /**
+     * Returns whether the current user can change/set all user attributes for
+     * the user being edited within the given data source, regardless of
+     * whether those attributes are already explicitly associated with that
+     * user.
+     *
+     * @param {String} [dataSource]
```

Comment:
Wait ... it *is* used here. It's not used in the other controllers though. Did you mean those?